radlab-ui/webapp/.env.test

```diff
@@ -0,0 +1,26 @@
+NEXT_PUBLIC_FIREBASE_PUBLIC_API_KEY=
+NEXT_PUBLIC_FIREBASE_AUTH_DOMAIN=
+NEXT_PUBLIC_FIREBASE_PROJECT_ID=
+NEXT_PUBLIC_FIREBASE_STORAGE_BUCKET=
+NEXT_PUBLIC_FIREBASE_MESSAGING_SENDER_ID=
+NEXT_PUBLIC_FIREBASE_APP_ID=
+NEXT_PUBLIC_FIREBASE_MEASUREMENT_ID=
+NEXT_PUBLIC_FIREBASE_DATABASE_URL
+
+NEXT_PUBLIC_GIT_URL=
+NEXT_PUBLIC_GIT_API_URL=
+NEXT_PUBLIC_GCP_SERVICE_ACCOUNT_EMAIL=
+NEXT_PUBLIC_GCP_PROJECT_ID=
+NEXT_PUBLIC_GCP_ORGANIZATION=
+NEXT_PUBLIC_PUBSUB_DEPLOYMENTS_TOPIC=
+NEXT_PUBLIC_PUBSUB_DEPLOYMENTS_DELETE_TOPIC=
+
+NEXT_PUBLIC_RAD_LAB_ADMIN_GROUP=rad-lab-admins
+NEXT_PUBLIC_RAD_LAB_USER_GROUP=rad-lab-users
+
+NEXT_PUBLIC_RAD_LAB_DOCS_SITE=https://googlecloudplatform.github.io/rad-lab/
+
+MODULE_DEPLOYMENT_BUCKET_NAME=
+
+GIT_TOKEN_SECRET_KEY_NAME=rad-lab-git-access-token
```

Comment:
This line should be removed as it contains a hardcoded reference to the name of the secret.

Reply:
The name of the secret is not sensitive (like the value of the secret is). I think this is an OK default since we are instructing folks in the documentation to name it the same.